test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -8,6 +8,11 @@ const path = require('path');
 const PLAYGROUND = path.resolve(__dirname, '../../../playground');
 const mocksFolder = `${PLAYGROUND}/mocks/data/idp/idx`;
 const mocksOauth2Folder = `${PLAYGROUND}/mocks/data/oauth2`;
+const noTranslateSelector = [
+  '.no-translate',
+  // hidden text for screen readers
```

Comment:
Should be removed after merging of https://github.com/okta/odyssey/pull/2074/files#diff-ba66395dd166ceaa1fb9be3e394dc3b399a088fdf32cbe51311b64f83f99ab8dR61